app/views/devise/passwords/edit.html.erb

```diff
@@ -12,12 +14,12 @@
     ) do |f| %>
   <%= f.input :reset_password_token, as: :hidden %>
   <%= f.full_error :reset_password_token %>
-  <%= render PasswordToggleComponent.new(
+  <%= render PasswordConfirmationComponent.new(
         form: f,
+        password_label: t('forms.passwords.edit.labels.password'),
         field_options: {
-          label: t('forms.passwords.edit.labels.password'),
-          required: true,
           input_html: {
+            aria: { describedby: 'password-description' },
             autocomplete: 'new-password',
```

Comment:
`PasswordConfirmationComponent` applies this by default, so we shouldn't need to assign it anymore.

```suggestion
```

https://github.com/18F/identity-idp/blob/a3a9cd0ec4d3b92b95b696d33ac91662dbe2ae02/app/components/password_confirmation_component.html.erb#L11